docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -1,79 +1,96 @@
 ---
 sidebar_navigation:
-  title: Nextcloud integration
+  title: Nextcloud integration setup
   priority: 600
-description: Nextcloud and OpenProject integration
+description: Nextcloud integration setup
 keywords: integrations, apps, Nextcloud
 ---
 
-# OpenProject and Nextcloud integration 
+# OpenProject and Nextcloud integration setup 
 
-The OpenProject and Nextcloud integration will improve the productivity of their enterprise users. It combines the strengths of the market leading content collaboration platform Nextcloud and the leading open source project management software OpenProject.
+The integration between OpenProject and Nextcloud has the potential to improve the productivity of all users. It combines the strengths of Nextcloud, the world’s most deployed on-premises collaborative file storage platform and OpenProject, the leading free and open source project management and collaboration software
 
-The integration is available starting with Nextcloud 20. It enables users to keep an eye on ongoing project activities directly in their Nextcloud instance.
+This integration allows you to link files and folders in Nextcloud with work packages in OpenProject, allowing you to see all files related to a work package (in Open Project) and all work packages related to a file (in Nextcloud). As a project member, you no longer need to lose time trying to find the right files to be able to complete your tasks, and the integration of OpenProject-specific notifications on Nextcloud dashboard ensures that no change goes unnoticed. 
 
-## Benefits of the integration
+## Step-by-step setup instructions
 
-The first step of the combined effort is the integration of OpenProject in the Nextcloud dashboard. Users can add an OpenProject widget to display latest changes to project's work packages. With this, it offers users a view of ongoing projects and activities.
+For integrating with OpenProject 12.2, the minimum required version of Nextcloud is 22. For the desktop Nextcloud application, the minimum supported version is 2.0.0. To be able to configure this integration, you need to have administrator privileges in both your Nextcloud and OpenProject instances.
```

Comment:
```suggestion
For integrating with OpenProject 12.2, the minimum required version of Nextcloud is 22. For the Nextcloud app  "OpenProject integration", that extends your Nextcloud installation, the minimum supported version is 2.0.0. To be able to configure this integration, you need to have administrator privileges in both your Nextcloud and OpenProject instances.
```

Reply:
Ah, good point @wielinde. I would remove the comma after 

> integration",

Looks like I can modify your suggested change, let me try it.